modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/asset/model/DepotAssetRendererFactoryWrapper.java

```diff
@@ -41,13 +45,19 @@ public DepotAssetRendererFactoryWrapper(
 		DepotApplicationController depotApplicationController,
 		DepotEntryLocalService depotEntryLocalService,
 		GroupLocalService groupLocalService,
-		SiteConnectedGroupGroupProvider siteConnectedGroupGroupProvider) {
+		SiteConnectedGroupGroupProvider siteConnectedGroupGroupProvider,
+		LayoutPrototypeLocalService layoutPrototypeLocalService,
+		LayoutPageTemplateEntryLocalService
+			layoutPageTemplateEntryLocalService) {
```

Comment:
These need to be sorted alphabetically (LayoutPage* < LayoutPrototype*).

Reply:
Done